### PR TITLE
18249 inplace editor multiple values preparation

### PR DESCRIPTION
**Pull request by 0xF013**
[`* `#18249` inplace editor multiple values preparement`](http://community.openproject.org/work_packages/18249)
- [x] remove https://github.com/opf/openproject/pull/2478/files#r24005719

### CODE REVIEW

**Comment by TeatroIO** [comment]
I've prepared a stage to preview changes. <a href='http://0xf013-feature-18249-inplace-editor-multiple-values-preparement.opf-openproject-2af22bf12c7dfdad4ddf.ttrcloud.com' target='_blank'>Open stage</a> or <a href='http://0xf013-feature-18249-inplace-editor-multiple-values-preparement.opf-openproject-2af22bf12c7dfdad4ddf.ttrcloud.com/teatro.log' target='_blank'>view logs</a>.

File: frontend/app/ui_components/inplace-editor-directive.js [hunk 1]
```diff
@@ -122,32 +129,48 @@ module.exports = function($timeout, FocusHelper, PathHelper, InplaceEditorDispat
...
+      OverviewTabInplaceEditorConfig.registerActiveEditorScope($scope);
     }
 
     function submit(notify) {
       // angular.copy here to make a new object instead of a reference
-      var data = angular.copy($scope.entity.form.embedded.payload.props);
-      InplaceEditorDispatcher.dispatchHook($scope, 'submit', data);
       $scope.isBusy = true;
+      var data = angular.copy($scope.entity.form.embedded.payload.props);
+      //$scope.collectChanges(data);
```
Comment (ulferts) [inline frontend/app/ui_components/inplace-editor-directive.js]: ??

Comment (0xF013) [inline frontend/app/ui_components/inplace-editor-directive.js]: @ulferts oops

**Comment by ulferts** [comment]
When I start to edit multiple values at once (>= 4) the frontend freezes (Chrome). Additionally, the saving of the values to the backend is not reliable. Out of the edited values, oftentimes only one is actually saved.

As for displaying the errors, we should definitely improve this (in a separate PR if more convenient) so that only the errors that belong to the attribute are displayed at the attribute.

**Comment by 0xF013** [comment]
@ulferts thanks for the review

**Comment by 0xF013** [comment]
@ulferts saving should be fixed now

**Comment by 0xF013** [comment]
karma is failing, will fix later today

**Comment by 0xF013** [comment]
@ulferts fixed

**Comment by ulferts** [comment]
@0xF013 looks good functionality-wise. 

I'd like to have somebody take a look at the code in more detail before it is merge (e.g. @floriank, @myabc), but to my unknowing eyes, it is looking good.

The only thing that might be missing is an integration test, but again, what do I know.

**Comment by floriank** [comment]
Looks good to me, I added a minor complaint due to commented code.
From a functionality perspective, this seems fine.

**Comment by 0xF013** [comment]
@floriank that comment has been removed by a later commit or amend.

btw I rebased it against dev

**Pull request merged**